modules/workers/locals.tf

```diff
@@ -36,6 +36,7 @@ locals {
     eviction_grace_duration      = 300
     force_node_delete            = true
     extended_metadata            = {} # empty pool-specific default
+    ignore_size                  = false
```

Comment:
You are hardcoding this value. Shouldn't you have a variable to allow the user to choose?

Reply:
something like pool-scaling-mode = autoscaler, terraform, manual

Even though manual is similar to non-autoscale, from a user perspective, it's better I think to indicate one as such. This then lets scale terraform-managed workers through terraform only

Reply:
I'm using nodepool defaults to set this parameter to avoid the impact on the existing customers.
At the same time, we need to have this control at the nodepool level, we can't use global variables.